### DISCUSSION TITLE
Jupiter notebook does not work on VScode. It works itself

### DISCUSSION THREAD

**Discussion by danling-northwestern** [microsoft/vscode-jupyter#13022] [Questions and Answers]
The error message looks like:

Failed to start the Kernel. 
Jupyter server crashed. Unable to connect. 
Error code from Jupyter: 1
Traceback (most recent call last):
  File "/Users/test/Library/Python/3.10/bin/jupyter-notebook", line 5, in 
    from notebook.notebookapp import main
ModuleNotFoundError: No module named 'notebook'. 
View Jupyter log for further details.

**Comment by DonJayamanne** [comment]
@danling-northwestern

Please could you enable logging as follows:
* Open settings & go into `Jupyter->Logging`
* Change the value to `verbose`
* Reload VS Code,
* Attempt to repro the issue & then copy the output from the `Jupyter` output panel.

**Reply by danling-northwestern** [reply]
It does not work... Same error message. Thank you though

**Reply by DonJayamanne** [reply]
> Same error message. Thank you though

Sorry, you misunderstood the comments.
the request was for more logs 
please could you enable logging as described above and send all of the logs
I might be able to figure out what is going on, 
the logs contain a lot of crucial information that is required to help you
* OS
* Version of vscode
* version of python and jupyter extensiohn
* version and path to python executables used to start jupyter and run a cell
* contains more error information
